virtual-setup/rstudio-login.md

```diff
@@ -1 +1,55 @@
-## RStudio Server instructions
\ No newline at end of file
+## RStudio Server Set Up
+
+We will send you your Username and temporary password by a direct message on Slack, after you are logged in on [Cancer Data Science](http://ccdatalab.org/slack).
+
+Go to https://rstudio.ccdatalab.org (You may want to bookmark this) and type in your username and temporary password, and click `Sign in`.
```

Comment:
Reiterate what RStudio is and what RStudio server is here - I don't think you can assume folks have looked at `intro-to-R-tidyverse/00a-rstudio_guide.md` before they get to this document.